axolotl-web/src/assets/dark.scss

```diff
@@ -29,6 +29,7 @@ $message-input-border: $body-bg;
 
 $sending-url: "sending_dark.svg";
 $sent-url: "check-circle-outline_dark.svg";
-$received-url: "double-check_dark.svg";
+$received-url: "double-check.svg";
+$receipt-url: "double-check_dark.svg";
```

Comment:
I would prefer to use "read" instead of "receipt". It isn't a verb in english, if I am not mistaking.